cdap-standalone/pom.xml

```diff
@@ -764,26 +699,6 @@
                   </resources>
                 </configuration>
               </execution>
-              <execution>
-                <id>copy-program-report-app</id>
```

Comment:
Is this intended to be deleted?

Reply:
Yes, this copies resources for spark2. We have a similar step for spark3 already present.